docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -19,117 +19,137 @@ Custom add-on packs allow you to deploy Kubernetes applications in clusters and
+- An SSH key available in the region where you plan to deploy the cluster.
+- [Docker Desktop](https://docs.docker.com/get-docker/) installed on your local machine to start the tutorial container. 
+- Basic knowledge of Docker containers, Kubernetes manifest file attributes, and Terraform.
 
+If you choose to use an OCI registry, you will need the following item.
+<!--- - An active OCI registry such as [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/) or [Harbor](https://goharbor.io/).-->
+- An active OCI registry such as [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/).
 
-4. An SSH key created in the region where you will deploy the cluster.
+If you opt for an ECR OCI registry, you will require the following.
 
+- An AWS Identity and Access Management (IAM) user with the following permissions.
 
-5. [Docker Desktop](https://docs.docker.com/get-docker/) installed on your local machine to start the tutorials container. 
+  - `ecr:CreateRepository`
+  - `ecr:InitiateLayerUpload`
+  - `ecr:CompleteLayerUpload`
+  - `ecr:InitiateLayerUpload`
+  - `ecr:PutImage`
+  - `ecr:UploadLayerPart`
+  - `ecr:BatchCheckLayerAvailability`
+  - `ecr:ListImages`
+  - `ecr:DescribeImages`
 
+:::caution
 
-6. Basic knowledge of Docker containers and Kubernetes manifest file attributes.
-
+There could be AWS expenses associated with this tutorial. After completing the tutorial, make sure to delete the infrastructure to avoid additional charges.
 
+:::
 
 
 ## Set Up the Tutorial Environment
 
-You will work in a Docker container pre-configured with the necessary tools for this tutorial. However, you can practice this tutorial in any `linux/amd64` or `x86_64` environment by installing the [necessary tools](https://github.com/spectrocloud/tutorials/blob/main/docs/docker.md#docker) and cloning the [GitHub repository](https://github.com/spectrocloud/tutorials/) that contains the tutorial files. Here are the steps to start the tutorials container. 
-<br />
+In this tutorial, you will work in a Docker container pre-configured with the necessary tools. Alternatively, you can choose to practice the tutorial in any `linux/amd64` or `x86_64` environment by installing the [required tools](https://github.com/spectrocloud/tutorials/blob/main/docs/docker.md#docker) and cloning the [GitHub repository](https://github.com/spectrocloud/tutorials/) that contains the tutorial files. To initialize the tutorial container, follow the steps described below. 
 
-Start the Docker Desktop on your local machine and ensure the daemon is available by issuing a command to list the currently active containers.
+Start Docker Desktop on your local machine and ensure that the Docker daemon is available by issuing a command to list the currently active containers.
```

Comment:
Should we add a short sentence on what the learner will see if the daemon is not started? I think it's worthwhile, since without this, they are dead in the water.

Reply:
Done!